operator/hack/lokistack_dev.yaml

```diff
@@ -5,6 +5,7 @@ metadata:
 spec:
   size: 1x.extra-small
   storage:
+    schemaVersion: v12
```

Comment:
Since we use the `+kubebuilder:default` annotation we should not need this here right?